<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>859</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_login_ttjj">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>登录天天基金</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_sync">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>同步持仓</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_search">
          <property name="placeholderText">
           <string>基金代码, 名称</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="table_fund">
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>基金代码</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>基金名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>昨日净值</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>估算净值</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>估算增长率</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>估算偏差</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>持仓成本价</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>持仓金额</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>预估收益</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total_money">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>持仓总金额: 200000元</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_assess">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2012-02-02 估算收益为10000元, 恭喜吃肉!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
